opensciencegrid/ospool-cm/ospool-overview/utils/glidein-summary.sh

```diff
@@ -15,11 +15,6 @@ echo "Sites with Singularity detected:"
 echo
 condor_status -const HAS_SINGULARITY -af GLIDEIN_ResourceName | sed 's/ /_/g' | sort | uniq -c | sort -n -r | perl -ane 'printf("    %-30s %5d\n", $F[1], $F[0])'
 
-echo
-echo "Singularity version counts:"
-echo
-condor_status -const HAS_SINGULARITY -af OSG_SINGULARITY_VERSION | sort | uniq -c
-
```

Comment:
Why get rid of this?

Reply:
Gotcha, in the future could you split off different changes into separate commits? It'll make parsing the git log a little friendlier in the future